ch2/fizzBuzz.js

```diff
@@ -0,0 +1,8 @@
+for(let i = 1; i <= 100; i++) {
+  console.log(
+    !(i % 3) && !(i % 5) ? "FizzBuzz" :
+    !(i % 3) ? "Fizz" :
+    !(i % 5) ? "Buzz" :
+    i
+    );
+}
```

Comment:
your code works, so i won't ask you to add tests, since i don't want to bog you down, but moving forward, please add tests!

Reply:
Wilco. Acknowledged